libqtile/backend/wayland/core.py

```diff
@@ -587,8 +647,20 @@ def _on_cursor_button(self, _listener: Listener, event: pointer.PointerButtonEve
             handled = self._process_cursor_button(button, pressed)
 
         if not handled:
+            if self._pressed_button_count == 1 and found:
+                assert self._implicit_grab is None
```

Comment:
Was this needed? Im surprised mypy wouldn't be able to infer this must be the case beyond the if statement on line 636

Reply:
Good point.  I originally had that line above line 636, but now it's automatic.